orttraining/orttraining/test/python/orttraining_test_ortmodule_execution_agents.py

```diff
@@ -1,18 +1,17 @@
-import numpy as np
-from onnx import TensorProto
-from onnx import helper
-
+from onnx import helper, TensorProto
+import pytest
 import torch
 
 import onnxruntime
-
 from onnxruntime.training.ortmodule._execution_agent import (
     InferenceAgent,
     TrainingAgent,
 )
+from _test_helpers import assert_values_are_close # pylint: disable=wrong-import-order
 
 
-def make_inference_model():
+@pytest.fixture
```

Comment:
did you test this trick on pytest 2.8.7 by running pytest -sv orttraining/orttraining/test/python/orttraining_test_ortmodule_execution_agents.py directly?

Reply:
I will try

Reply:
Yes it worked (on CPU)
```
================================================= test session starts ==================================================
platform linux -- Python 3.6.9, pytest-2.8.7, py-1.9.0, pluggy-0.3.1 -- /usr/bin/python
cachedir: ../../../.cache
rootdir: /home/ryoto/onnxruntime, inifile:
collecting 0 items2021-07-15 05:40:25.551692361 [E:onnxruntime:Default, provider_bridge_ort.cc:950 Ensure] Failed to load library libonnxruntime_providers_shared.so with error: libonnxruntime_providers_shared.so: cannot open shared object file: No such file or directory
2021-07-15 05:40:25.551752662 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:1688 CreatePybindStateModule] Init provider bridge failed.
No CUDA runtime is found, using CUDA_HOME='/usr/local/cuda'
collected 2 items

orttraining_test_ortmodule_execution_agents.py::test_onnx_graph_forward PASSED
orttraining_test_ortmodule_execution_agents.py::test_onnx_graph_forward_backward PASSED

=============================================== 2 passed in 1.43 seconds ===============================================
```

Is `Init provider bridge failed` something we should worry about?